docs/source/dev_guide/release.rst

```diff
@@ -73,21 +78,29 @@ Releasing on GitHub
 
    * You can scroll through `zppy commits <https://github.com/E3SM-Project/zppy/commits/main>`_ for a list of changes.
 
-5. If this version is a release candidate (``<version>`` appended with ``rc``), checkmark `This is a pre-release`.
-6. Click `Publish release`.
-7. CI/CD release workflow is automatically triggered.
+5. Click `Publish release`.
+6. CI/CD release workflow is automatically triggered.
 
-Releasing Anaconda
+Releasing on conda-forge: release candidates
 ------------------
 
-1. Be sure to have already completed :ref:`Releasing On GitHub <github-release>`. This triggers the CI/CD workflow that handles Anaconda releases.
-2. Wait until the CI/CD build is successful. You can view all workflows at `All Workflows <https://github.com/E3SM-Project/zppy/actions>`_.
-3. Check the https://anaconda.org/e3sm/zppy page to view the newly updated package.
+1. Make a PR to `conda-forge <https://github.com/conda-forge/zppy-feedstock/>`_ from your fork of the feedstock. Note that the conda-forge bot does not work for release candidates.
 
-   * Release candidates are assigned the ``e3sm_dev`` label
-   * Production releases are assigned the ``main`` label
+   * Start from the current dev branch and update the version number and the sha256 sum manually.
+   * Set the build number back to 0 if needed.
+   * Make the dev branch the target of the PR. Then, the package build on conda-forge will end up with the ``e3sm_dev`` label.
```

Comment:
I think this section will definitely need clarification, but we can improve the wording when we work on the next release together.